SIL.Windows.Forms/ImageToolbox/PalasoImage.cs

```diff
@@ -288,10 +288,53 @@ public static PalasoImage FromFile(string path)
 				Metadata = Metadata.FromFile(path),
 				_tempFilePath = tempPath
 			};
+			NormalizeImageOrientation(i.Image);
```

Comment:
So, do we really need to Hard-Rotate the image? Would just preserving the exif information work?